### DISCUSSION TITLE
Overlay not loading

### DISCUSSION THREAD

**Discussion by doctornoodlearms** [AvaloniaUI/Avalonia#9030] [Q&A] [answered]
A mod for a game I play creates an overlay using Avalonia but the overlay isn't able to load. I posted an issue on the mods repository a couple days ago and haven't gotten a response. I was hoping someone in here might be able to help.

In the logs there is a line that says: [Info   :   Console] Overlay will not load, because the binary's path is not configured

I suspect that the message is from Avalonia but I'm not sure.

**Comment by timunie** [comment, marked as answer]
Sorry to say, but I think it's not an Avalonia issue. Avalonia is just the UI you see.

Guess: Sounds like a setting in your App is missing.
